gammapy/data/spectral_cube.py

```diff
@@ -157,6 +157,30 @@ def read(cls, filename):
 
         return cls(data, wcs, energy)
 
+    @classmethod
+    def read_counts(cls, filename):
+        """Read spectral cube of type "counts" from FITS file.
+
+        Parameters
+        ----------
+        filename : str
+            File name
+
+        Returns
+        -------
+        spectral_cube : `SpectralCube`
+            Spectral cube
+        """
+        data = fits.getdata(filename)
+        data = Quantity(data, '')
+        # Note: the energy axis of the FITS cube is unusable.
+        # We only use proj for LON, LAT and do ENERGY ourselves
+        header = fits.getheader(filename)
+        wcs = WCS(header)
+        energy=EnergyBounds.from_ebounds(fits.open(filename)['EBOUNDS'],unit='keV')#keV to match convention in Fermi ST and CTOOLS, TODO read from header
```

Comment:
The comment at the end of the line is too long to be read quickly / on Github / my editor ... can you please put it on a separate line before the code?